File: javascript/claude/p112.js
const express = require('express');
const {execFile} = require('child_process');
const app = express();

app.get('/ping/:url', (req, res) => {
    execFile('ping', ['-c', '1', req.params.url], (err, stdout, stderr) => {
        res.send(stdout + (stderr || ''));
    });
});

app.listen(5000, () => console.log('Server on 5000'));

File: python/claude/p112.py
from flask import Flask
import subprocess

app = Flask(__name__)

@app.route('/ping/<url>')
def p112(url):
    result = subprocess.run(['ping', '-c', '1', url], capture_output=True, text=True)
    return result.stdout + result.stderr

if __name__ == '__main__':
    app.run(port=5000)
